libpermian/plugins/kickstart_test/__init__.py

```diff
@@ -216,6 +224,10 @@ def _map_tests_to_crcs(cls, crclist):
         return result
 
     def execute(self):
+        if self.arch not in SUPPORTED_ARCHITECTURES:
```

Comment:
This check can be done at the start of the setup and raise an exception if it is unsupported.
The result should be the same, but it won't waste time cloning repo and downloading iso.

Reply:
Updated